File: leetcode-submissions/1960-check-if-the-sentence-is-pangram/solution.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:

        alphabets = {letter : 0 for letter in range(97, 123)}

        for char in sentence:
            ascii_code = ord(char)
            alphabets[ascii_code] += 1
    
        
        return min(alphabets.values()) > 0
        

File: leetcode-submissions/1960-check-if-the-sentence-is-pangram/solution.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        //set and check if the size is 26?
        
        Set<Character> chars = new HashSet<>();
        for(int i = 0; i < sentence.length(); i++) {
            chars.add(sentence.charAt(i));
        }
        return chars.size() == 26;
        
    }
}
